examples/src/main/kotlin/usageExamples/bulkWrite/BulkWrite.kt

```diff
@@ -0,0 +1,65 @@
+package usageExamples.bulkWrite
+
+// :replace-start: {
+//    "terms": {
+//       "CONNECTION_URI_PLACEHOLDER": "\"<connection string uri>\"",
+//       "import io.github.cdimascio.dotenv.dotenv\n": ""
+//    }
+// }
+// :snippet-start: bulk-write-usage-example
+import com.mongodb.MongoException
+import com.mongodb.client.model.*
```

Comment:
import * is generally frowned upon as explicit imports are better than implicit ones. Star imports aren't used in the java examples.

Has it been agreed that its ok for Kotlin docs?

Reply:
Good catch. It hasn't been discussed, but we've been operating under the assumption that we're matching the Java docs as much as possible. I'll disable IntelliJ's import optimization and will update the code and snippet to use the explicit imports/match the Java example

Reply:
Updated